set.mm

```diff
@@ -485557,6 +485580,16 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  $( A sub-division-ring of a field is itself a field, so it is a subfield.  We
+     can therefore use ` SubDRing ` to express subfields.  (Contributed by
+     Thierry Arnoux, 11-Jan-2025.) $)
+  fldsdrgfld $p |- ( ( F e. Field /\ A e. ( SubDRing ` F ) )
```

Comment:
This is important/of general interest enough to be moved to main immediately.

Reply:
perhaps df-sdrg could link to this theorem: 'If the overring is commutative this is a field ( ~ fldsdrgfld )'

Reply:
Moved!

Reply:
> perhaps df-sdrg could link to this theorem

Linked!